opm/patch_manager.py

```diff
@@ -219,8 +225,10 @@ def mine_patches(self, output_directory, config, output_csv=None):
         else:
             csv_filename = output_csv
         output = open(csv_filename, "a")
+        if self.save_subjectID:
+            output.write("SubjectID,")
         if self.label_map is not None:
-            output.write("Slide Patch path, Label Map Patch path, Label Map Result\n")
+            output.write("Slide Patch path, Label Map Patch path, Patch Composition\n")
```

Comment:
Removing spaces from CSV, because it could cause parsing issues in downstream analysis.

```suggestion
            output.write("SlidePatchPath,LabelMapPatchPath,PatchComposition\n") 
```